tools/cloudflow-sbt-plugin/src/main/scala/cloudflow/sbt/CloudflowLocalRunnerPlugin.scala

```diff
@@ -348,6 +364,12 @@ object CloudflowLocalRunnerPlugin extends AutoPlugin {
     println(GraphLayout.renderGraph(graph))
   }
 
+  def getAppLayout(connections: List[(String, String)]): String = {
```

Comment:
can you remove `printAppLayout` and move the `println` to the call sites?

Reply:
Done!